lecture_1/cli_args.js

```diff
@@ -1,26 +1,46 @@
-const fs = require("fs");
 const { stat } = require("fs/promises");
+const yargs = require("yargs/yargs");
+const { hideBin } = require("yargs/helpers");
 
-const filePath = process.argv[2];
+const PATH_FLAG = "path";
+const ENTITY_TYPE = "entity-type";
 
-console.log(`Getting stats of ${filePath}`);
+const argv = yargs(hideBin(process.argv)).argv;
 
-// fs.stat(filePath, (err, stats) => {
-//   if (err) {
-//     throw err;
-//   }
+if (!argv[PATH_FLAG]) {
+  console.log("Error: path is not defined");
+  process.exit();
```

Comment:
Be careful with process exit https://nodejs.org/api/process.html#process_process_exit_code
Don't forget about exit code, it's 0 by default (success)